pages/common/vault.md

```diff
@@ -0,0 +1,27 @@
+# vault
+
+> A CLI to interact with HashiCorp Vault.
+
+- Create a new vault, requiring at least 2 out of 5 keyshares to unseal:
+
+`vault init -key-shares={{5}} -key-threshold={{2}}`
+
+- Unseal the vault by providing one of the keyshares. Repeat with necessary key-shares until unsealed:
+
+`vault unseal {{key-share-x}}`
+
+- Authenticate client against vault, using an authentication token:
+
+`vault auth {{authentication-token}}`
+
+- Store a new secret in the vault using the generic back-end called secret:
```

Comment:
Please correct me if I'm reading this wrong, but is "secret" the name of the generic/default back-end? If so, maybe add quote marks around it to make that clearer, and perhaps a comma before "called".